docs/guide/design.md

```diff
@@ -59,28 +59,28 @@ ezpublish:
 
 This is what individual keys in the configuration mean:
 
-- **`ezpublish`** and **`system`** are obligatory at the start of any configuration file which defines views.
-- **`default`** defines the siteaccess for which the configuration will be used. "default", as the name suggests, determines what views are used when no other configuration is chosen. You can also have separate keys defining views for other siteaccesses.
-- **`user`** and **`layout`** point to the main template file that is used in any situation where no other template is defined. All other templates extend this one. See [below](#page-layout) for more information.
-- **`content_view`** defines the view provider.
+- `ezpublish` and `system` are obligatory at the start of any configuration file which defines views.
+- `default` defines the SiteAccess for which the configuration will be used. "default", as the name suggests, determines what views are used when no other configuration is chosen. You can also have separate keys defining views for other SiteAccesses.
+- `user` and `layout` point to the main template file that is used in any situation where no other template is defined. All other templates extend this one. See [below](#page-layout) for more information.
+- `content_view` defines the view provider.
 
 !!! note
 
-    In earlier versions of eZ CMS, `location_view` was used as the view provider. It is now deprecated.
+    In earlier version `location_view` was used as the view provider. It is now deprecated.
```

Comment:
I suggest adding "It is deprecated since the version x.x"

Reply:
The problem is, it is deprecated since kernel 6.0, so in truth since Platform 1.0, and I believe we shouldn't attempt to track all changes reaching back to 5.x in this doc - there are simply too many.
Plus, all sections referring to `location_view` will likely change completely following https://jira.ez.no/browse/EZP-28086